src/screens/Landing/components/Authors.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { AUTHORS } from "../../../data/landingPageData";
+import { AuthorCard } from "./AuthorCard";
+
+export const Authors = () => {
+	return (
+		<>
+			<h1 className="main-header-author text-center">Featured authors</h1>
+			<div className="author-container">
+				<div className="card-set card-author">
+					{AUTHORS.map((authorData) => (
```

Comment:
why use all caps here? is it a convention

Reply:
Yes, for constants it's recommended to use all caps